spec/unit/appmgmt_spec.rb

```diff
@@ -46,7 +46,7 @@ def compile_to_env_catalog(string)
 
       application app {
         prod { one: host => ahost, export => Cap[cap] }
-        cons { two: host => ahost, consume => Cap[cap] }
```

Comment:
What I meant was, if I add the {{host => ahost}} declaration in a test that in all other aspects is equal to the one on line 49, then what would that actually test? That a parameter can be assigned? We know it can and there are several other tests that verifies it. What we want to test here is that the lack of such assignment doesn't cause a problem.